.travis.yml

```diff
@@ -7,11 +7,19 @@ env:
   - DIST=trusty
   global:
     # 20170821: as of writing there is no 'latest' (hashicorp/packer/issues/5265)
-  - PACKER_ZIP="packer_1.2.1_linux_amd64.zip"
-  - PACKER_URL="https://releases.hashicorp.com/packer/1.2.1/packer_1.2.1_linux_amd64.zip"
+  - PACKER_URL="echo https://releases.hashicorp.com/packer/$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')/packer_$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')_linux_amd64.zip"
```

Comment:
Would it not make more sense to put this into a shell sript and call that from the ```.travis.yml```?

Reply:
I am fine to have it in the .travis.yml. This way we have everything regarding testing in one file.